docs/Installation/Installation.md

```diff
@@ -6,19 +6,121 @@ See LICENSE.txt for details.
 
 \tableofcontents
 
-This page details the installation procedure for SpECTRE on personal computers
-using x86-64 processors. For configuring SpECTRE, please refer to the
```

Comment:
I think we should keep a blurb of some kind here at the top before going in to details. The import links I think that should be highlighted are

- installation on clusters
- versioning and releases